src/mpid/ch4/netmod/ofi/ofi_win.h

```diff
@@ -47,8 +47,8 @@ static inline int MPIDI_OFI_win_allgather(MPIR_Win * win, void *base, int disp_u
 
     bits_for_context_id = total_bits_avail -
         MPIDI_Global.max_windows_bits - MPIDI_Global.max_huge_rma_bits;
-    max_contexts_allowed = 1 << (bits_for_context_id);
-    max_instances_allowed = 1 << (bits_for_instance_id);
+    max_contexts_allowed = (size_t) 1 << (bits_for_context_id);
```

Comment:
Wow, good catch. For some reason I thought the variable type was `size_t`. I'll fix it that way.